examples/pytorch/rgcn/experimental/README.md

```diff
@@ -1,62 +1,64 @@
 ## Distributed training
 
-This is an example of training RGCN node classification in a distributed fashion. Currently, the example train RGCN graphs with input node features. The current implementation follows ../rgcn/entity_claasify_mp.py.
+This is an example of training RGCN node classification in a distributed fashion. Currently, the example only support training RGCN graphs with no input features. The current implementation follows [../rgcn/entity_claasify_mp.py](../rgcn/entity_claasify_mp.py).
 
-Before training, please install some python libs by pip:
+Before training, install python libs by pip:
 
 ```bash
-sudo pip3 install ogb
-sudo pip3 install pyinstrument
-sudo pip3 install pyarrow
+pip3 install ogb pyinstrument
 ```
 
-To use the example and tools provided by DGL, please clone the DGL Github repository.
-
-```bash
-git clone --recursive https://github.com/dmlc/dgl.git
-```
-Below, we assume the repository is cloned in the home directory.
-
 To train RGCN, it has four steps:
 
-### Step 0: set IP configuration file.
+### Step 0: Setup a Distributed File System 
+* You may skip this step if your cluster already has folder(s) synchronized across machines. 
 
-User need to set their own IP configuration file before training. For example, if we have four machines in current cluster, the IP configuration
-could like this:
+To perform distributed training, files and codes need to be accessed across multiple machines. A distributed file system would perfectly handle the job (i.e., NFS, Ceph). 
 
+Here is an example of how to setup NFS. First, install essential libs on the storage server
 ```bash
-172.31.0.1
-172.31.0.2
-172.31.0.3
-172.31.0.4
-```
+sudo apt-get install nfs-kernel-server
+``` 
 
-Users need to make sure that the master node (node-0) has right permission to ssh to all the other nodes.
+Then create a folder for sharing and (optional) remove all restrictions
+```bash
+sudo mkdir -p /NFS
+# Change ownership and remove restrictions
+sudo chown -R nobody:nogroup /NFS
+sudo chmod 755 /NFS
```

Comment:
does this require root permission to write `/NFS`? I tried it and didn't find a way to write data to the folder with a regular account.

Reply:
Usually NFS setup is performed by an admin account, so I assume that the user has sudo access. 

It is possible to use a common directory, says a folder under user's home, as a NFS point (without sudo access). However, in this case, they will not be able to edit `/etc/exports` and `/etc/fstab` to enable NFS.

Reply:
NFS setup should be done by admin. but when using NFS (e.g., writing data to NFS), we need to allow a normal user account. otherwise, it'll be very inconvenient.

we should show an example of using a folder under user's home directory. NFS setup can be done by admin. It's fine.